apps/studio-next/src/helpers/driver.ts

```diff
@@ -0,0 +1,111 @@
+import { driver } from "driver.js";
+import "driver.js/dist/driver.css";
+
+export const driverObj = driver({
+  popoverClass: "driverjs-theme",
+  showProgress: true,
+  showButtons: ["next", "previous", "close"],
+  nextBtnText: "Next",
+  prevBtnText: "Back",
+
+  steps: [
+    {
+      element: "#tour-start",
+      popover: {
+        title: "Welcome to AsyncAPI Studio",
+        description:
+          "Discover a powerful tool for designing, documenting, and managing AsyncAPI-based applications. This tour will guide you through key features to enhance your AsyncAPI development workflow.",
+      },
+    },
+    {
+      element: "#sidebar",
+      popover: {
+        title: "Control Center",
+        description:
+          "This control center allows you to toggle the editor, information panel, and HTML preview on or off. It's also your gateway to creating new AsyncAPI templates for various protocols like Apache Kafka, WebSocket, HTTP, and more. Customize your workspace and jumpstart your AsyncAPI design process from here.",
+        side: "left",
+        align: "start",
+      },
+    },
+    {
+      element: "#information",
+      popover: {
+        title: "Information Panel",
```

Comment:
IMO it's better to call this one `Navigation Panel` and adjust the id's accordingly. It helps user navigate the AsyncAPI document right?

Reply:
Yes, right.